config/component_macros.cmake

```diff
@@ -712,8 +712,8 @@ function( copy_dll_link_libraries_to_build_dir target )
           get_target_property( isimp ${lib} IMPORTED )
           if(isimp)
             get_target_property( link_libs3 ${lib} INTERFACE_LINK_LIBRARIES)
-          else()
-            get_target_property( link_libs2 ${lib} LINK_LIBRARIES )
+          #else()
+          #  get_target_property( link_libs2 ${lib} LINK_LIBRARIES )
           endif()
           list( APPEND link_libs ${link_libs2} )
```

Comment:
Is `link_libs2` defined at this point?

Reply:
@timmah Let me rebase before you review this PR.  This above change was necessary for cmake-3.17.  But I might remove it from this PR.